### PR TITLE
Update FAQs_Pricing.md

### PR DESCRIPTION
**Pull request by bfsantos**
FAQ update: combination of usage and perpetual and clarifications

### CODE REVIEW

File: dataminer-overview/FAQ/FAQs_Pricing.md [hunk 1]
```diff
@@ -2,28 +2,43 @@
...
+### Can different commercial models exist in the same organization?
 
-  Yes, through an optional service subscription.
+Yes. Licensing and usage are measured at the DMS (DataMiner System) level. An organization can have perpetual licenses on one DMS while running usage-based services on a different DMS.
 
-- **Can I use Professional Services credits in exchange for DataMiner credits?**
+### Can different commercial models exist in the same DataMiner System?
 
-  No.
+Yes, but limited. If a DMS has a perpetual license, it can only use "DataMiner.Services" and "STaaS" from the usage services.
```
Comment (MariekeGO) [inline dataminer-overview/FAQ/FAQs_Pricing.md:32]: Is "DataMiner.Services" indeed the spelling that should be used here? Usually, this is spelled as "dataminer.services".

File: dataminer-overview/FAQ/FAQs_Pricing.md [hunk 2]
```diff
@@ -2,28 +2,43 @@
...
+
+### Can I transition my perpetual licenses into a subscription?
+
+Yes. Get in touch with your Account Manager.
```
Comment (MariekeGO) [inline dataminer-overview/FAQ/FAQs_Pricing.md:36]: This comes across as a little unfriendly. Could we rephrase it like this?
```suggestion
Yes. Your Account Manager will be able to provide more information and assist you with this.
```

**Review by MariekeGO — COMMENTED** [review]
The capitalization in this pull request is quite inconsistent, with "Commercial models" and "commercial models", and "Deploy models" and "deploy models" both being used. As the rest of the documentation does not use a capital letter in this case, and the capital letter also looks quite strange here, I'd prefer to use lower case in all cases here. Would this be OK for you?

File: dataminer-overview/FAQ/FAQs_Pricing.md (hunk 1 above)
Comment (bfsantos) [reply dataminer-overview/FAQ/FAQs_Pricing.md:32]: you're right, should be dataminer.services

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**